src/v3/src/transformer/layout/securityQuestion/securityQuestionVerify.ts

```diff
@@ -35,20 +34,15 @@ export const transformSecurityQuestionVerify: IdxStepTransformer = ({ transactio
     },
   };
 
-  const questionElement: DescriptionElement = {
-    type: 'Description',
-    noMargin: true,
-    options: {
-      content: relatesTo?.value?.profile?.question as string,
-      variant: 'h6',
-    },
-  };
```

Comment:
This was using a paragraph tag as a label for the security question verify field. This should use the actual label to resolve a11y issues and match parity with v2.